website/www/site/layouts/index.html

```diff
@@ -71,13 +71,16 @@ <h1>Try Beam Playground</h1>
   <a class="playground__mobile" href="https://play.beam.apache.org/">
     <img src="images/playground.png" alt="beam playground">
   </a>
-  <iframe
-          src="https://play.beam.apache.org/embedded?enabled=true&example=SDK_JAVA/PRECOMPILED_OBJECT_TYPE_EXAMPLE/MinimalWordCount&code="
-          width="100%"
-          height="700px"
-          class="code-snippet playground"
-          allow="clipboard-write">
-  </iframe>
+  <div id="playgroundIframeContainer">
+    <iframe
```

Comment:
Let's format this file. Do it in a separate commit in case other reviewers ask us to revert the change.

Reply:
was told to make less commits. before that i made a new one on each logical step.

Reply:
@bullet03 in this case it would make sense to commit the formatted code separately